### PR TITLE
Warn if cairo-test plugin is missing

### PR DESCRIPTION
**Pull request by maciektr**
commit-id:9ff328fc

### CODE REVIEW

File: extensions/scarb-cairo-test/src/main.rs [hunk 1]
```diff
@@ -43,6 +44,7 @@ fn main() -> Result<()> {
     let metadata = MetadataCommand::new().inherit_stderr().exec()?;
 
     check_scarb_version(&metadata);
+    check_cairo_test_plugin(&metadata);
```
Comment (THenry14) [inline extensions/scarb-cairo-test/src/main.rs:47]: when will this be shown?
I suppose it should only when someone tries to execute a test and does not have any plugin configured? or what is the deal?

Comment (maciektr) [reply extensions/scarb-cairo-test/src/main.rs:47]: Only if you run `cairo-test` (this is the main file of the cairo-test binary distributed with Scarb) and you miss the plugin.

Comment (THenry14) [reply extensions/scarb-cairo-test/src/main.rs:47]: aaaah, sorry misread the file name. thanks for pointing this out :+1:

**Review by THenry14 — APPROVED** [review]

**Review by piotmag769 — COMMENTED** [review]
Gj

**Pull request merged**